packages/devtools-source-map/jest-setup.js

```diff
@@ -0,0 +1,5 @@
+// TODO: Remove this file after upgrading to Jest 22.4.1+
+// More info: https://github.com/facebook/jest/issues/687
+console.debug = console.log
```

Comment:
In server environments, source-map calls `console.debug`, which does not exist in the version of Jest that this repo is currently running. Some other problems prevented me from simply upgrading Jest. I didn't think it was the right time to be going down that rabbithole.